client/packages/glsp-sprotty/src/features/execute/di.config.ts

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2018 EclipseSource Services GmbH.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *   
+ * Contributors:
+ * 	Tobias Ortmayr - initial API and implementation
+ ******************************************************************************/
+import { ContainerModule } from "inversify";
+import { TYPES } from "sprotty/lib";
+import { ExecuteCommandMouseListener } from "./execute-command";
+
+const executeCommandModule = new ContainerModule(bind => {
+    bind(TYPES.MouseListener).to(ExecuteCommandMouseListener);
```

Comment:
I'm wondering whether a double-click should invoke a command by default. I guess in most cases this will depend on the graphical language, could be a button, a link, Click with modifier, etc. A specific client could easily hook up its own listeners and invoke the ExecuteCommand. However, once we bind the double lick listener, clients won't be able to get rid of this listener easily. So this might be something that is rather bound by the specific client, isn't it?

Reply:
The modules are not added per default. Each specific client decides which feature-modules should be added to the sprotty di container. So the commandExecutonModule can easily be omitted or replaced.

see  also https://github.com/eclipsesource/graphical-lsp/blob/a2d16ed72f4c3ddf23b8064e8e5e5c87561a47d2/client/packages/workflow-sprotty/src/di.config.ts#L53

Of course we can extend the capabilites of the module later on to also react to button clicks etc. similiar to how it is handled in the expandModule  https://github.com/theia-ide/sprotty/blob/6ae2599560c24319a2ecc23c4ba8b0bdbb4258b4/client/src/features/expand/di.config.ts#L13